docs/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ Runtime
 Client
 ------
 
+* Remove OnOffenceHandler::can_report (#8414)
```

Comment:
```suggestion
```

This is not a client change, but good that you considered updating a changelog. 

Unfortunately given the sheer volume of changes within frame between each release, we don't keep track of changelogs per-pallet (except for a few exceptional ones -- e.g. `elections-phragmen`). You could initiate one for `Offence` individually if you want.